src/io_hid_keys.rs

```diff
@@ -0,0 +1,15 @@
+// exports from <IOKit/hid/IOHIDKeys.h>
+
+use libc::c_int;
+
+pub type IOHIDReportType = c_int;
+pub const kIOHIDReportTypeInput: IOHIDReportType   = 0;
+pub const kIOHIDReportTypeOutput: IOHIDReportType  = 1;
+pub const kIOHIDReportTypeFeature: IOHIDReportType = 2;
+pub const kIOHIDReportTypeCount: IOHIDReportType   = 3;
+
+pub type IOHIDManagerOptions = c_int;
```

Comment:
I see this definition in `IOKit/hid/IOHIDManager.h`. I may not have the latest SDK, so just let me know if this has changed since the version I'm looking at.